documentation/observability/5-deploy-to-kyma.md

```diff
@@ -69,14 +69,14 @@ To create the SAP Cloud Logging service instance, follow these steps:
     ```
     To apply the configuration, use the following command:
     ```
-    kubectl apply -n sap-cloud-logging-integration -f clsbinding.yaml
+    kubectl apply -n sap-cloud-logging-integration -f cloud-logging-binding.yaml
     ```
     
-7. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://kyma-project.io/#/telemetry-manager/user/integration/sap-cloud-logging/README?id=ship-logs-to-sap-cloud-logging).
+7. Ship the logs to the SAP Cloud Logging service. See [Ship Logs to SAP Cloud Logging](https://help.sap.com/docs/btp/sap-business-technology-platform/integrate-with-sap-cloud-logging?ship-logs-to-sap-cloud-logging).
```

Comment:
It's rendered correctly, but in the raw version it looks weird to jump from step 5 (line 54) to 7 (line 75).

Reply:
Please check once more, I changed ordering